### ISSUE TITLE
Not sound with rtcpMuxPolicy

### ISSUE THREAD

**Issue by esvills** [onsip/SIP.js#480]
```js
let options = {
sessionDescriptionHandlerOptions: {
                        constraints: {
                            audio: true,
                            video: false
                        },
                        peerConnectionOptions:{
                             rtcConfiguration:{
                                 rtcpMuxPolicy: 'negotiate'
                             }
                         }
                    },
                };
            session.accept(options);

            session.sessionDescriptionHandler.on("userMedia", function (stream) {
                document.getElementById("local").srcObject = stream;
            });
            session.sessionDescriptionHandler.on("addStream", function (streamEvent) {
                console.log("GOT STREAM", streamEvent);
                let domElement = document.getElementById("remote");
                domElement.srcObject = streamEvent.stream;
                domElement.play();
                session.sessionDescriptionHandler.peerConnection.getRemoteStreams()[0].onaddtrack = function (trackEvent) {
                    console.log("adding track", trackEvent);
                };
            });
````
Chrome v 62.0.3202.94
mozilla 56.0.2
MacOs

**Comment by egreenmachine** [comment]
Can you please clean this up and include logs? Thanks

**Comment by jlaine** [comment]
Also it would help to know what kind of server you are talking to, and why you need to force rtcpMuxPolicy to "negotiate".

**Comment by egreenmachine** [comment]
I think that this is an issue with your Asterisk instance. The call is not up for more than 10 seconds before a `BYE` is received. 

SIP.js does not do anything with that setting. It is simply passed down to the underlying WebRTC library.

**Comment by jlaine** [comment]
I see you are passing the peerConnectionOptions inside the call to accept(), and believe this is what's causing issues : it results in tearing down / recreating the peer connection. Could you please try passing the peerConnectionOptions at UA creation time by using sessionDescriptionHandlerFactoryOptions and report back?

**Comment by jlaine** [comment]
Eric : I think we should add logging messages inside initPeerConnection. It would allow us to see at what point the peer connection is (re)created and would make debugging easier.

**Comment by egreenmachine** [comment]
@jlaine I believe you are probably correct in that the peer connection is getting torn down. Logging should be added when it is torn down and when it is created. I will try and update the documentation to represent that too.

**Comment by jlaine** [comment]
@esvills Could you please retest with SIP.js 0.9.1 and the change suggested in https://github.com/onsip/SIP.js/issues/480#issuecomment-346767350 ?

**Comment by egreenmachine** [comment]
I think that this ticket was resolved with the 0.9.1 release. If there is no response in a week, I am going to close this due to inactivity.

**Comment by jlaine** [comment]
I'd vote to close this issue, no response from the reporter

**Comment by egreenmachine** [comment]
Closing as we think that this is resolved in 0.9.1+

**Comment by bharath-inference** [comment]
Hi, I have the same issue on my side. It would be great if you could reopen the ticket. As @jlaine  suggested, I have passed the peerConnectionOptions at UA creation time by using sessionDescriptionHandlerFactoryOptions but still, there is no sound. Posted the logs below for your reference. Thanks 

SIP.logs

sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | configuration parameters after validation:
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · viaHost: "v7edc70aal4v.invalid"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · uri: sip:1658@10.216.235.71
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · wsServers: [{"ws_uri":"wss://10.216.235.71:4433","sip_uri":"<sip:10.216.235.71:4433;transport=ws;lr>","weight":0,"status":0,"scheme":"WSS"}]
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · custom: {}
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · displayName: "WebRTC App"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · password: NOT SHOWN
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · registerExpires: 600
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · register: true
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · registrarServer: sip:10.216.235.71
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · wsServerMaxReconnection: 3
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · wsServerReconnectionTimeout: 4
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · connectionRecoveryMinInterval: 2
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · connectionRecoveryMaxInterval: 30
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · keepAliveInterval: 0
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · extraSupported: []
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · usePreloadedRoute: false
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · userAgentString: "SIP.js/0.9.1"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · noAnswerTimeout: 60000
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · traceSip: true
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · hackViaTcp: false
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · hackIpInContact: false
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · hackWssInTransport: false
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · hackAllowUnregisteredOptionTags: false
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · sessionDescriptionHandlerFactoryOptions: {"peerConnectionOptions":{"rtcConfiguration":{"rtcpMuxPolicy":"negotiate"}}}
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · contactName: "omdjibg8"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · contactTransport: "ws"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · forceRport: false
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · autostart: true
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · autostop: true
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · rel100: "none"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · replaces: "none"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · sessionDescriptionHandlerFactory: function defaultFactory(session, options) {
    return new SessionDescriptionHandler(session, options);
  }
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · authenticationFactory: undefined
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · allowLegacyNotifications: false
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · allowOutOfDialogRefers: false
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · authorizationUser: "1000"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · instanceId: "d00293d3-13f7-4744-b4e3-64c564726036"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · sipjsId: "t19oc"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | · hostportParams: "10.216.235.71"
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | user requested startup...
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.transport | connecting to WebSocket wss://10.216.235.71:4433
callControl-9.1.js:86 UA {log: LoggerFactory, logger: Logger, cache: {…}, configuration: {…}, dialogs: {…}, …}
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.transport | WebSocket wss://10.216.235.71:4433 connected
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.ua | connection state set to 0
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.transport | sending WebSocket message:

REGISTER sip:10.216.235.71 SIP/2.0
Via: SIP/2.0/WSS v7edc70aal4v.invalid;branch=z9hG4bK6350980
Max-Forwards: 70
To: "WebRTC App" <sip:1658@10.216.235.71>
From: "WebRTC App" <sip:1658@10.216.235.71>;tag=b1et5kllj9
Call-ID: pvqnopnmo0uirvnls6tsrj
CSeq: 8193 REGISTER
Contact: <sip:omdjibg8@v7edc70aal4v.invalid;transport=ws>;reg-id=1;+sip.instance="<urn:uuid:d00293d3-13f7-4744-b4e3-64c564726036>";expires=600
Allow: ACK,CANCEL,INVITE,MESSAGE,BYE,OPTIONS,INFO,NOTIFY,REFER
Supported: path, gruu, outbound
User-Agent: SIP.js/0.9.1
Content-Length: 0



sip-0.9.1.js:807 Thu Mar 08 2018 10:31:12 GMT+1100 (AUS Eastern Summer Time) | sip.transport | received WebSocket text message:

SIP/2.0 200 OK
Via: SIP/2.0/WSS v7edc70aal4v.invalid;received=10.203.1.192;branch=z9hG4bK6350980
To: "WebRTC App" <sip:1658@10.216.235.71>;tag=79f170d9eedc1092ba7849ac2aa01fc0.d011
From: "WebRTC App" <sip:1658@10.216.235.71>;tag=b1et5kllj9
Call-ID: pvqnopnmo0uirvnls6tsrj
CSeq: 8193 REGISTER
Contact: <sip:omdjibg8@10.203.1.192:54906;transport=ws>;expires=600;received="sip:10.203.1.192:54906;transport=WSS"
Server: OpenSIPS (2.3.3 (x86_64/linux))
Content-Length: 0



callControl-9.1.js:127 makecalltest MediaStream {id: "6b91df2d-c0d1-4fa2-b4cd-ed024a40f815", active: false, onaddtrack: null, onremovetrack: null, onactive: null, …}
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | initPeerConnection
sip-0.9.1.js:8962 [Deprecation] The rtcpMuxPolicy option is being considered for removal and may be removed no earlier than M62, around October 2017. If you depend on it, please see https://www.chromestatus.com/features/5654810086866944 for more details.
initPeerConnection @ sip-0.9.1.js:8962
SessionDescriptionHandler @ sip-0.9.1.js:8650
defaultFactory @ sip-0.9.1.js:8663
invite @ sip-0.9.1.js:5854
(anonymous) @ sip-0.9.1.js:7323
Promise.then (async)
UA.invite @ sip-0.9.1.js:7322
makeCalltest @ callControl-9.1.js:138
onclick @ dtmfindex.html?_ijt=5i6t0s21cfh4h9t15tu430ukm8:63
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | New peer connection created
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | acquiring local media
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | acquired local media streams
sip-0.9.1.js:807 RTCSessionDescription {type: "offer", sdp: "v=0
↵o=- 8896635696832632558 2 IN IP4 127.0.0.1
↵s…2460 label:2bb02ddb-4e38-4046-a636-af571b187a51
↵"}
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | RTCIceGatheringState changed: gathering
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:2346683772 1 udp 2122255103 2001::9d38:6abd:86d:353e:ce00:3509 64456 typ host generation 0 ufrag vs64 network-id 2 network-cost 50
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:3732548911 1 udp 2122194687 10.203.1.192 64457 typ host generation 0 ufrag vs64 network-id 1
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:2346683772 2 udp 2122255102 2001::9d38:6abd:86d:353e:ce00:3509 64458 typ host generation 0 ufrag vs64 network-id 2 network-cost 50
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:3732548911 2 udp 2122194686 10.203.1.192 64459 typ host generation 0 ufrag vs64 network-id 1
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:3311169932 1 tcp 1518275327 2001::9d38:6abd:86d:353e:ce00:3509 9 typ host tcptype active generation 0 ufrag vs64 network-id 2 network-cost 50
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:2432291295 1 tcp 1518214911 10.203.1.192 9 typ host tcptype active generation 0 ufrag vs64 network-id 1
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:3311169932 2 tcp 1518275326 2001::9d38:6abd:86d:353e:ce00:3509 9 typ host tcptype active generation 0 ufrag vs64 network-id 2 network-cost 50
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:2432291295 2 tcp 1518214910 10.203.1.192 9 typ host tcptype active generation 0 ufrag vs64 network-id 1
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:678057211 1 udp 1685987071 49.255.202.246 64457 typ srflx raddr 10.203.1.192 rport 64457 generation 0 ufrag vs64 network-id 1
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | ICE candidate received: candidate:678057211 2 udp 1685987070 49.255.202.246 64459 typ srflx raddr 10.203.1.192 rport 64459 generation 0 ufrag vs64 network-id 1
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | RTCIceGatheringState changed: complete
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.transport | sending WebSocket message:

INVITE sip:0370103322@10.216.235.71 SIP/2.0
Via: SIP/2.0/WSS v7edc70aal4v.invalid;branch=z9hG4bK3207670
Max-Forwards: 70
To: <sip:0370103322@10.216.235.71>
From: "WebRTC App" <sip:1658@10.216.235.71>;tag=pibstdoubc
Call-ID: t19oc03qgvie2lsi389f
CSeq: 6530 INVITE
Contact: <sip:omdjibg8@v7edc70aal4v.invalid;transport=ws;ob>
Allow: ACK,CANCEL,INVITE,MESSAGE,BYE,OPTIONS,INFO,NOTIFY,REFER
Supported: outbound
User-Agent: SIP.js/0.9.1
Content-Type: application/sdp
Content-Length: 2485

v=0
o=- 8896635696832632558 2 IN IP4 127.0.0.1
s=-
t=0 0
a=group:BUNDLE audio
a=msid-semantic: WMS vyVUBCWG0LYQPtca9gvkJ0mt0FExHzwRyL2m
m=audio 64457 UDP/TLS/RTP/SAVPF 111 103 104 9 0 8 106 105 13 110 112 113 126
c=IN IP4 49.255.202.246
a=rtcp:64459 IN IP4 49.255.202.246
a=candidate:2346683772 1 udp 2122255103 2001::9d38:6abd:86d:353e:ce00:3509 64456 typ host generation 0 network-id 2 network-cost 50
a=candidate:3732548911 1 udp 2122194687 10.203.1.192 64457 typ host generation 0 network-id 1
a=candidate:2346683772 2 udp 2122255102 2001::9d38:6abd:86d:353e:ce00:3509 64458 typ host generation 0 network-id 2 network-cost 50
a=candidate:3732548911 2 udp 2122194686 10.203.1.192 64459 typ host generation 0 network-id 1
a=candidate:3311169932 1 tcp 1518275327 2001::9d38:6abd:86d:353e:ce00:3509 9 typ host tcptype active generation 0 network-id 2 network-cost 50
a=candidate:2432291295 1 tcp 1518214911 10.203.1.192 9 typ host tcptype active generation 0 network-id 1
a=candidate:3311169932 2 tcp 1518275326 2001::9d38:6abd:86d:353e:ce00:3509 9 typ host tcptype active generation 0 network-id 2 network-cost 50
a=candidate:2432291295 2 tcp 1518214910 10.203.1.192 9 typ host tcptype active generation 0 network-id 1
a=candidate:678057211 1 udp 1685987071 49.255.202.246 64457 typ srflx raddr 10.203.1.192 rport 64457 generation 0 network-id 1
a=candidate:678057211 2 udp 1685987070 49.255.202.246 64459 typ srflx raddr 10.203.1.192 rport 64459 generation 0 network-id 1
a=ice-ufrag:vs64
a=ice-pwd:5sQtDu+nhGAjMxFdK09JyERa
a=ice-options:trickle
a=fingerprint:sha-256 9F:7E:A4:00:7D:B1:AE:EF:68:26:F3:C0:43:27:AE:D1:DC:60:1B:36:10:44:E9:FC:A6:03:1B:CC:5B:EF:E1:90
a=setup:actpass
a=mid:audio
a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
a=sendrecv
a=rtcp-mux
a=rtpmap:111 opus/48000/2
a=rtcp-fb:111 transport-cc
a=fmtp:111 minptime=10;useinbandfec=1
a=rtpmap:103 ISAC/16000
a=rtpmap:104 ISAC/32000
a=rtpmap:9 G722/8000
a=rtpmap:0 PCMU/8000
a=rtpmap:8 PCMA/8000
a=rtpmap:106 CN/32000
a=rtpmap:105 CN/16000
a=rtpmap:13 CN/8000
a=rtpmap:110 telephone-event/48000
a=rtpmap:112 telephone-event/32000
a=rtpmap:113 telephone-event/16000
a=rtpmap:126 telephone-event/8000
a=ssrc:2987782460 cname:tXL8XaTqUmXZqiik
a=ssrc:2987782460 msid:vyVUBCWG0LYQPtca9gvkJ0mt0FExHzwRyL2m 2bb02ddb-4e38-4046-a636-af571b187a51
a=ssrc:2987782460 mslabel:vyVUBCWG0LYQPtca9gvkJ0mt0FExHzwRyL2m
a=ssrc:2987782460 label:2bb02ddb-4e38-4046-a636-af571b187a51


sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.transport | received WebSocket text message:

SIP/2.0 100 Giving a try
Via: SIP/2.0/WSS v7edc70aal4v.invalid;received=10.203.1.192;branch=z9hG4bK3207670
To: <sip:0370103322@10.216.235.71>
From: "WebRTC App" <sip:1658@10.216.235.71>;tag=pibstdoubc
Call-ID: t19oc03qgvie2lsi389f
CSeq: 6530 INVITE
Server: OpenSIPS (2.3.3 (x86_64/linux))
Content-Length: 0



sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.transport | received WebSocket text message:

SIP/2.0 200 OK
Via: SIP/2.0/WSS v7edc70aal4v.invalid;received=10.203.1.192;branch=z9hG4bK3207670
Record-Route: <sip:10.216.235.71;r2=on;lr>
Record-Route: <sip:10.216.235.71:4433;transport=wss;r2=on;lr>
From: "WebRTC App" <sip:1658@10.216.235.71>;tag=pibstdoubc
To: <sip:0370103322@10.216.235.71>;tag=920229403
Call-ID: t19oc03qgvie2lsi389f
CSeq: 6530 INVITE
Contact: <sip:10.216.235.38:5060>
Content-Type: application/sdp
Allow: INVITE, ACK, BYE, INFO, OPTIONS, CANCEL
Content-Length: 501

v=0
o=BroadWks 5402 0 IN IP4 10.216.235.38
s=Media Server SDP
t=0 0
m=audio 19808 UDP/TLS/RTP/SAVPF 0
c=IN IP4 10.216.235.71
a=rtpmap:0 PCMU/8000
a=ptime:20
a=sendrecv
a=rtcp:19809
a=setup:passive
a=fingerprint:sha-1 64:CC:6F:CB:04:E8:47:AA:2C:70:EA:B3:74:50:93:A9:2D:F1:25:D8
a=ice-ufrag:0Ph7IDGc
a=ice-pwd:YVoUwHnqDPSbpp2PUQLZFnIGbX
a=candidate:fjKCoL9XjVgdLwxF 1 UDP 2130706431 10.216.235.71 19808 typ host
a=candidate:fjKCoL9XjVgdLwxF 2 UDP 2130706430 10.216.235.71 19809 typ host


sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.dialog | new UAC dialog created with status CONFIRMED
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | Using deprecated stream API
LoggerFactory.print @ sip-0.9.1.js:807
LoggerFactory.(anonymous function) @ sip-0.9.1.js:824
Logger.(anonymous function) @ sip-0.9.1.js:818
peerConnection.onaddstream @ sip-0.9.1.js:8973
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | stream added
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.invitecontext.sessionDescriptionHandler | track added
sip-0.9.1.js:807 Thu Mar 08 2018 10:31:15 GMT+1100 (AUS Eastern Summer Time) | sip.transport | sending WebSocket message:

ACK sip:10.216.235.38:5060 SIP/2.0
Route: <sip:10.216.235.71:4433;transport=wss;r2=on;lr>
Route: <sip:10.216.235.71;r2=on;lr>
Via: SIP/2.0/WSS v7edc70aal4v.invalid;branch=z9hG4bK4639079
Max-Forwards: 70
To: <sip:0370103322@10.216.235.71>;tag=920229403
From: "WebRTC App" <sip:1658@10.216.235.71>;tag=pibstdoubc
Call-ID: t19oc03qgvie2lsi389f
CSeq: 6530 ACK
Supported: outbound
User-Agent: SIP.js/0.9.1
Content-Length: 0
